.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  extends: ['@vertexvis/vertexvis-typescript', 'plugin:promise/recommended'],
```

Comment:
`plugin:promise/recommended` might be a nice addition to our our internal eslint plugin.